src/config.ts

```diff
@@ -115,8 +124,8 @@ export const getFhirConfig = async (): Promise<FhirConfig> => ({
             oauthPolicy: {
                 authorizationEndpoint: `${patientPickerEndpoint}/authorize`,
```

Comment:
No using the fhirworks-auth-issuer-endpoint parameter value here will mean that we need to provide it in the patientPicketEndpoint option on deploy, which we may not have.

Reply:
This should be patientPickerEndpoint, because this is for authorization code flow. The issuer endpoint is  https://id-sbx.resmed.com, but the authorization endpoint is  https://uxuj41rmw1.execute-api.us-west-2.amazonaws.com/dev/authorize

That's why patientPicketEndpoint option  is required and not optional

Reply:
I just pushed a commit. Please Let me know if I haven't covered your comment